<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceAddWindow</class>
 <widget class="QDialog" name="ServiceAddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>315</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>214</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>315</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add component</string>
  </property>
  <widget class="QGroupBox" name="comp_add_gBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>281</width>
     <height>240</height>
    </rect>
   </property>
   <property name="title">
    <string>Component Info</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>28</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="service_name_hLayout">
       <item>
        <widget class="QLabel" name="name_label">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="component_name_lEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="specif_label">
         <property name="text">
          <string>Specifications:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="spec_lEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="name_equipment_hLayout">
       <item>
        <widget class="QLabel" name="component_type_label">
         <property name="text">
          <string>Component type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="component_type_comboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="rel_date_l">
         <property name="text">
          <string>Release date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="release_date_dEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="price_hLayout">
       <item>
        <widget class="QLabel" name="price_label">
         <property name="text">
          <string>Price:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="price_dSpinBox">
         <property name="maximum">
          <double>999999999999.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>281</width>
     <height>40</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="add_comp_btn">
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>11</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Accept</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
